src/containerapp/azext_containerapp/tests/latest/test_containerapp_session_code_interpreter.py

```diff
@@ -51,58 +51,65 @@ def test_containerapp_session_code_interpreter_e2e(self, resource_group):
             identifier_name,
             code),
             checks=[
-            JMESPathCheck('properties.status', 'Success'),
-            JMESPathCheck('properties.stdout', 'Hello world\n')
+            JMESPathCheck('status', 'Succeeded'),
```

Comment:
Hi @yitaopan 

When I was running test [test_containerapp_session_code_interpreter_e2e](https://github.com/Azure/azure-cli-extensions/blob/2af456d98196cec532b7afee7598ad4571c4a7a3/src/containerapp/azext_containerapp/tests/latest/test_containerapp_session_code_interpreter.py#L20)，[test_containerapp_session_code_interpreter_nodelts_e2e](https://github.com/Azure/azure-cli-extensions/blob/2af456d98196cec532b7afee7598ad4571c4a7a3/src/containerapp/azext_containerapp/tests/latest/test_containerapp_session_code_interpreter_nodelts.py#L20) in CLI, they failed with error：Unauthorized
 
repro(tested on northcentralusstage and eastus regions)：
az containerapp sessionpool create -g xinyueastusrepro -n xinyusessiontesteastus --cooldown-period  300 --location eastus
az containerapp session code-interpreter execute  -g xinyueastusrepro -n xinyusessiontesteastus --identifier testidentifierafwasfdsaf --debug --code 'print(0)'
 
Got：Unauthorized
 
https://eastus.dynamicsessions.io/subscriptions/23f95f0e-e782-47be-9f97-56035ec10e42/resourceGroups/xinyueastusrepro/sessionPools/xinyusessiontesteastus/code/execute?identifier=testidentifierafwasfdsaf&api-version=2024-10-02-preview
{'properties': {'identifier': 'testidentifierafwasfdsaf', 'codeInputType': 'inline', 'executionType': 'synchronous', 'code': 'print(0)', 'timeoutInSeconds': 60}}
Unauthorized
 
cli.azure.cli.core.util: Request URL: '[https://eastus.dynamicsessions.io/subscriptions/23f95f0e-e782-47be-9f97-56035ec10e42/resourceGroups/xinyueastusrepro/sessionPools/xinyusessiontesteastus/code/execute?identifier=testidentifierafwasfdsaf&api-version=2024-10-02-preview'](https://eastus.dynamicsessions.io/subscriptions/23f95f0e-e782-47be-9f97-56035ec10e42/resourceGroups/xinyueastusrepro/sessionPools/xinyusessiontesteastus/code/execute?identifier=testidentifierafwasfdsaf&api-version=2024-10-02-preview%27)
cli.azure.cli.core.util: Request method: 'POST'
cli.azure.cli.core.util: Request headers:
cli.azure.cli.core.util:     'User-Agent': 'python/3.10.11 (Windows-10-10.0.26100-SP0) AZURECLI/2.66.0 (PIP)'
cli.azure.cli.core.util:     'Accept-Encoding': 'gzip, deflate'
cli.azure.cli.core.util:     'Accept': '*/*'
cli.azure.cli.core.util:     'Connection': 'keep-alive'
cli.azure.cli.core.util:     'x-ms-client-request-id': 'e68b5152-142f-49e4-9dc8-c8ad22dc9133'
cli.azure.cli.core.util:     'Content-Type': 'application/json'
cli.azure.cli.core.util:     'CommandName': 'containerapp session code-interpreter execute'
cli.azure.cli.core.util:     'ParameterSetName': '-g -n --identifier --debug --code'
cli.azure.cli.core.util:     'Authorization': 'Bearer eyJ0eXAiOiJKV...'
cli.azure.cli.core.util:     'Content-Length': '161'
cli.azure.cli.core.util: Request body:

Reply:
Did you do something relate to role assignment before running the test?